src/main/java/com/kodilla/ecommercee/mapper/OrderMapper.java

```diff
@@ -4,25 +4,22 @@
 import com.kodilla.ecommercee.dto.OrderDto;
 import org.springframework.stereotype.Component;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Component
 public class OrderMapper {
 
-    public Order mapToOrder(final OrderDto orderDto) {
-        //return new Order(orderDto.getId(), orderDto.getDate(), orderDto.isCompleted(), orderDto.getProductList(), orderDto.getUserId());
-        return null;
+    public Order mapToOrder(OrderDto orderDto) {
+        return new Order(orderDto.getId(),orderDto.getDate(),orderDto.isCompleted(),orderDto.getUserId());
```

Comment:
Wydaje mi się że potrzebna będzie relacja Order User w związku z czym  Pole  userId powinno zostać zamienione na encje User.  Napisałem również w innym pull requescie (JDP190701-14) 
Jeśli ktoś przychyli sie do mojej sugestii  tutaj też będą potrzebne zmiany.  Co Ty o tym sądzisz?

Reply:
No tak , masz rację to może zajme sie tym . Zrobie relację oraz zmienię mapper.